ui.go

```diff
@@ -293,7 +293,7 @@ func fileInfo(f *file, d *dir) string {
 	for _, s := range getInfo(d.path) {
 		switch s {
 		case "size":
-			if !(f.IsDir() && gOpts.dircounts) {
+			if !f.IsDir() || !gOpts.dircounts {
```

Comment:
Applying De Morgan's law to sometimes only need to check one value

Reply:
I think it would be better to reorder the code so that the `dircounts` code is handled first:

```go
if f.IsDir() && gOpts.dircounts {
    // handle dircounts
    continue
}

// handle normal logic
```

Reply:
That reads much nicer, good catch!